presto-main/src/test/java/com/facebook/presto/geospatial/TestGeoFunctions.java

```diff
@@ -1371,4 +1371,13 @@ private void assertInvalidGeometryJson(String json)
     {
         assertInvalidFunction("geometry_from_geojson('" + json + "')", "Invalid GeoJSON:.*");
     }
+
+    @Test
+    public void testGMapsPolyline()
```

Comment:
Two test cases seems sparse.  Are you sure this is sufficient test coverage?  If we look at polyline libraries, how extensive are their test cases and is there anything we can learn from them?

Reply:
I tend to agree. Ordinarily I would just grab some actual trip data and use that to form test cases, but on the public internet that reveals PII.

Instead I've created a case manually - a case with 

(1) precision in the input floats higher than supported
(2) duplicate points in a row
(3) line crosses back over itself (not very significant, but hey)
(4) line terminates at the origin (non consecutive duplicate points)